sunpy/timeseries/tests/test_timeseries_factory.py

```diff
@@ -150,6 +150,21 @@ def test_implicit_new_goes16(self):
         ts_goes = sunpy.timeseries.TimeSeries(new_goes16_filepath)
         assert isinstance(ts_goes, sunpy.timeseries.sources.goes.XRSTimeSeries)
 
+    def test_implicit_goes_satno(self):
+        # Test a GOES TimeSeries for satellite number
+        ts_goes = sunpy.timeseries.TimeSeries(goes_filepath)
+        assert ts_goes.satellite_number == 15
+
+    def test_implicit_new_goes15_satno(self):
+        # Test a GOES TimeSeries for satellite number
+        ts_goes = sunpy.timeseries.TimeSeries(new_goes15_filepath)
+        assert ts_goes.satellite_number == 15
+
+    def test_implicit_new_goes16_satno(self):
+        # Test a GOES TimeSeries for satellite number
+        ts_goes = sunpy.timeseries.TimeSeries(new_goes16_filepath)
+        assert ts_goes.satellite_number == 17
```

Comment:
Is it expected that the satellite number is 17, even though (according to pre-exising code in sunpy!) this is a GOES 16 file?

Reply:
Yeah that is actually a goes 17 file despite the name, I noticed this when designing the test, was deciding whether to raise a separate issue but it really doesn't effect anything other than this new test.

`>>> goes=ts.TimeSeries(new_goes16_filepath)
>>> goes.satellite_number
17
>>> goes.meta.metas[0]['platform']
b'g17'
>>> goes.meta.metas[0]['id']
b'sci_xrsf-l2-flx1s_g17_d20201016_v2-0-1.nc'`